test/DurableTaskEndToEndTests.cs

```diff
@@ -232,7 +244,7 @@ public DurableTaskEndToEndTests(ITestOutputHelper output)
                     TestHelpers.AssertLogMessageSequence(
                         this.output,
                         this.loggerProvider,
-                        "HelloWorldOrchestration_Activity",
+                        logReplayEvents ? "HelloWorldOrchestration_Activity" : "HelloWorldOrchestration_Activity_Skip_Replay_Events",

```

Comment:
The `extendedSessions` boolean parameter already filters out replay events. Is there some common logic we can use so that we don't need to create multiple sets of expected log outputs for the same basic tests?

Reply:
Great point. I will work on it. Thank you!